broid-kik/src/core/Parser.ts

```diff
@@ -44,13 +44,13 @@ export class Parser {
     if (!normalized || R.isEmpty(normalized)) { return Promise.resolve(null); }
 
     const activitystreams = this.createActivityStream(normalized);
-    activitystreams.actor = {
+    activitystreams.target = {
```

Comment:
you can read the details about [actor](https://www.w3.org/TR/activitystreams-core/#actors) and the [target](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-target). Basically, target in the messaging context mean `to`